design/hardware-classification-controller/support-for-new-parameters-hwcc-DiskAndNIC.md

```diff
@@ -46,39 +52,50 @@ Add support for new parameters:
    Under `diskSelector`, new parameters `hctl` and `rotational` will be
    added. `diskSelector` will be declared as an array.
 
-   User can find multiple disk type combination example of `hctl` and
-   `rotational` parameters in rule book [Docs](https://github.com/metal3-io/hardware-classification-controller/tree/master/docs).
+   Example rules will be documented in the hardware-classification-controller
+   repository under docs.
+
+   For selection of NVMe disk, user can provide `rotational` parameter
```

Comment:
Are all SSDs NVMe?

Reply:
No, for clarification I modified the few lines. Hope it will clear.